package.json

```diff
@@ -17,7 +17,7 @@
     "docs:examples": "node tooling/docs/examples-to-md.js examples/node/*.js",
     "docs:api": "node apidoc.js",
     "lint": "eslint . --ext .ts,.js",
-    "test": "nyc mocha './test/unit/' './test/integration/'",
+    "test": "nyc mocha --require ts-node/register 'test/integration/*' './test/unit/*'",
```

Comment:
`ts-node/register` already specified in `.mocharc.js`, also I specified accepted extensions there, so `/*` also not needed